src/components/select/select.tsx

```diff
@@ -59,10 +59,14 @@ export class Select implements FormComponent {
 
   /**
    * Render a text input showing a list of suggested elements.
-
    */
   @Prop() suggest: boolean;
 
+  /**
+   * Specifies a short hint that describes the expected value of the field.
```

Comment:
```suggestion
   * Text that appears in the form control when it has no value set.
```

Reply:
I take the description from w3schools hehe.

Reply:
Yes, I thought it wasn't clearly explaining what the attribute thas (placeholder text doesn't need to be short nor has to show the expected value, even though it's often used that way) so I looked the one I proposed on MDN, which, BTW, is a much better quality and trustworthy resource than w3schools.

Reply:
Edited document in 7d60907